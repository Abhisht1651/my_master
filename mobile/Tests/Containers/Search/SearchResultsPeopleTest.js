// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import TestFile from '../../../App/Containers/Search/SearchResultsPeople'
import ListOfUsers from '../../../App/Components/ListOfUsers/ListOfUsers'

describe('Search Results People Screen', () => {
  it('should render the component itself correctly', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render the ListOfUsers Component', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper.find(ListOfUsers)).toHaveLength(11)
  })
})
