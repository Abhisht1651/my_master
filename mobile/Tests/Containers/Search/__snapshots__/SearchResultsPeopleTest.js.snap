// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Results People Screen should render the component itself correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ScrollViewMock
    showsVerticalScrollIndicator={false}
  >
    <View
      style={
        Object {
          "marginVertical": 20,
          "width": 650,
        }
      }
    >
      <View
        key="0"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="1"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="2"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="3"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="4"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="5"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="6"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="7"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="8"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="9"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
      <View
        key="10"
        style={
          Object {
            "marginBottom": 10,
          }
        }
      >
        <ListOfUsers
          buttonText="Add"
          buttonTextAfterPress="Added"
          image={
            Object {
              "default": "",
            }
          }
          name="Amelia Jones"
          needoptiontext={false}
          onCancelPress={[Function]}
          onPress={[Function]}
          optionalText="Following 54 Storylines"
          profileType={1}
          type={1}
        />
      </View>
    </View>
  </ScrollViewMock>
</View>
`;
